src/CMakeLists.txt

```diff
@@ -71,7 +71,7 @@ set_property(TARGET ${PROJECT_NAME} PROPERTY COMPILE_FLAGS ${LLVM_CPPFLAGS})
 execute_process(COMMAND ${LLVM_CONFIG} --includedir OUTPUT_VARIABLE LLVM_INCLUDES OUTPUT_STRIP_TRAILING_WHITESPACE)
 include_directories("${LLVM_INCLUDES}")
 
-set(LLVM_COMPONENTS "${LLVM_COMPONENTS}" core bitwriter x86 ipo)
```

Comment:
I removed `bitreader asmparser irreader mirparser` in a later commit since Halide doesn't have them. Do we need them for something?